.env.testing

```diff
@@ -7,11 +7,11 @@ APP_URL=http://localhost
 LOG_CHANNEL=stack
 
 DB_CONNECTION=pgsql
-DB_HOST=postgres
+DB_HOST=localhost
 DB_PORT=5432
 DB_DATABASE=scrapper-test
-DB_USERNAME=
-DB_PASSWORD=
+DB_USERNAME=carlos
```

Comment:
I think this file should not be versioned I think it was a mistake of mine, it is to handle the environment variables in the testing environment